src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -1,8 +1,13 @@
 package org.broadinstitute.dsde.rawls.model
 
+import org.broadinstitute.dsde.rawls.RawlsException
 import org.broadinstitute.dsde.rawls.model.WorkspaceAccessLevel.WorkspaceAccessLevel
 import org.joda.time.DateTime
 
+object Attributable {
+  val reservedAttributeNames = Set("name", "entityType")
```

Comment:
Workspace has entityType as a reserved property as well, but i guess that's fine since it isnt an entity anyway